frontend/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
Please update dependencies so they match the ones in RDC. The current one look like this:

```
    "peerDependencies": {
        "axios": "0.21.1",
        "react": "17.0.1",
        "react-dom": "17.0.1",
        "react-router-dom": "5.2.0",
        "styled-components": "5.2.1"
    },
```